Makefile

```diff
@@ -1,125 +1,133 @@
-GCR_URL = us.gcr.io/vcm-ml
-COMPILED_TAG_NAME ?= latest
-ENVIRONMENT_TAG_NAME ?= latest
-COMPILE_OPTION ?=
-COMPILE_TARGET ?= fv3gfs-compiled
-CUDA ?=n
-BUILD_ARGS ?=
+# setup (use XXX=<value> make <target> to override)
+GCR_URL                 ?= us.gcr.io/vcm-ml
+DOCKERFILE              ?= docker/Dockerfile
+COMPILED_TAG_NAME       ?= latest
```

Comment:
I appreciate the thought, but can we please use consistent whitespace conventions with what we have been using and use in every other repo/Makefile, at least unless we decide as a group to change it?

Reply:
IMO, we don't have any clearly established white space convention. I'm personally ok with changes in style like this in the absence of a CI enforced linting system, but they do clutter the diff.

Reply:
Disagree. Clearly more readable. Happy to change the other Makefiles as well in follow-on PRs.

Reply:
OK, I won't insist. My only opposition to this style of formatting is that it is hard to keep it up to date, especially when a new variable is introduced which is longer than the whitespace. I'd prefer this style if we had an auto-formatting tool which applies it.